### ISSUE TITLE
frequently cause ANR

### ISSUE THREAD

**Issue by gmfbilu** [Devlight/InfiniteCycleViewPager#45]
![s70606-16355549](https://cloud.githubusercontent.com/assets/12381195/26820741/7fd33a36-4ad6-11e7-99f6-338da06bac69.jpg)
![s70606-16363454](https://cloud.githubusercontent.com/assets/12381195/26820742/7fdb2a48-4ad6-11e7-9d0d-e74cf8d9374c.jpg)

**Comment by msr550** [comment]
Yes, ANR coming coming for rendering issue. It is taking more time render.

**Comment by DewAbhijeet** [comment]
Is there any solution to this issue?
